MANIFEST.in

```diff
@@ -1,2 +1,2 @@
 include README.rst
-include requirements.txt
+include requirements*.txt
```

Comment:
This will also include requirements-dev.txt. If that's fine then it looks good to me.